.circleci/config.yml

```diff
@@ -650,7 +650,7 @@ defaults:
   - job_b_ubu_ubsan_clang: &job_b_ubu_ubsan_clang
       <<: *workflow_trigger_on_tags
       name: b_ubu_ubsan_clang
-      cmake_options: -DSANITIZE=address
+      cmake_options: -DSANITIZE=undefined
```

Comment:
Oh yeah, I noticed that too and fixed in the CI cleanup PR https://github.com/ethereum/solidity/pull/13982/commits/06d37b7a240c3925dd8b051a422534576d91a71e as well haha

But yeah, I didn't know when it was introduced.

Reply:
It's a bug, so a separate PR would be a good idea anyway. That PR is supposed to only contain refactors :)